tests/common/mod.rs

```diff
@@ -106,20 +108,26 @@ pub fn gen_jwt(account_id: Option<&str>) -> (String, String) {
 /// nice error handling with `?` in the tests.
 #[allow(dead_code)]
 pub async fn initialize_sdk() -> Result<IronOxide, IronOxideErr> {
+    init_sdk_with_config(&Default::default()).await
+}
+
+pub async fn init_sdk_with_config(config: &IronOxideConfig) -> Result<IronOxide, IronOxideErr> {
     let account_id: UserId = create_id_all_classes("").try_into()?;
     IronOxide::user_create(
         &gen_jwt(Some(account_id.id())).0,
         USER_PASSWORD,
         &UserCreateOpts::new(false),
+        None,
     )
     .await?;
     let device = IronOxide::generate_new_device(
         &gen_jwt(Some(account_id.id())).0,
         USER_PASSWORD,
         &Default::default(),
+        None,
```

Comment:
Should this and line 120 use `config.sdk_operation_timeout` to keep the timeouts consistent?

Reply:
I agreed with this at first, but after looking at it more, the name says `init_sdk_with_config`, so I think the other timeouts for `user_create` and `generate_new_device` are separate concerns. I came to this conclusion after running a test that was expecting `initialize` to timeout after passing the config value, but once I reused the timeout from the config, `user_create` started timing out first.